db/migrate/20151015183729_update_miq_database_default_repo_name.rb

```diff
@@ -0,0 +1,33 @@
+class UpdateMiqDatabaseDefaultRepoName < ActiveRecord::Migration
+  class MiqDatabase < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    has_many :authentications, :as => :resource
```

Comment:
This needs to directly reference the class below otherwise it will load the "real" class, which we don't want.

Even so, polymorphic is really hard to do properly in migrations because the has_many tries to be too smart and ends up loading from the model anyway....might be better to manually query with `.where(:resource_type => 'MiqDatabase', :resource_id => db.id)`